### ISSUE TITLE
readme/docs for pyiterate

### ISSUE THREAD

**Issue by oxinabox** [JuliaPy/PyCall.jl#609]
I don't see any doc changes attached to #594

Can we have some, please?

**Comment by tkf** [comment]
It would be nice to use Documenter.jl. The docstring added in #594 would have included in the documentation semi-automatically if we were using it.

**Comment by stevengj** [comment]
Yes, when PyCall was created, neither Documenter.jl nor docstrings existed.

**Comment by zsz00** [comment]
I don't see any doc , where is the documenter of PyCall ???
